README.md

```diff
@@ -2,13 +2,13 @@
 
 # A Package for Exploratory Data Analysis (eda_analysis)
 
-![](https://github.com/sweber15/eda_analysis/workflows/build/badge.svg) 
+![build](https://github.com/UBC-MDS/edapython/workflows/build/badge.svg) 
```

Comment:
A lot of the badges look like they are failing- although this PR has all green checks. Any reason why?

Reply:
that's probably due to the master branch failing, since all the badges are status in the master branch.